shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/helpers/extensions/ios/AnyFlow.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.blisskmmic.helpers.extensions.ios
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+// For iOS Unit Test
+class AnyFlow<T>(source: Flow<T>): Flow<T> by source {
+
+    constructor(result: T) : this(flow { emit(result) })
```

Comment:
Cool! I did not think about this one. I was trying to mock the data with Swift, and it was not working :D